code/game/objects/items/twohanded.dm

```diff
@@ -485,6 +486,43 @@
 		desc = "A makeshift spear with [G] attached to it."
 	update_icon()
 
+//maul
+/obj/item/twohanded/maul
+	icon_state = "toolboxmaulr0"
+	lefthand_file = 'icons/mob/inhands/weapons/hammers_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/weapons/hammers_righthand.dmi'
+	name = "makeshift maul"
+	desc = "A haphazardly-constructed yet still deadly weapon of ancient design."
+	force = 10
```

Comment:
Why is this different than force unwielded?

Reply:
i fixed that in my next commit sorry